libsolidity/CompilerUtils.cpp

```diff
@@ -41,18 +41,22 @@ unsigned CompilerUtils::loadFromMemory(unsigned _offset, Type const& _type,
 	return loadFromMemoryHelper(_type, _fromCalldata, _padToWordBoundaries);
 }
 
-void CompilerUtils::loadFromMemoryDynamic(Type const& _type, bool _fromCalldata, bool _padToWordBoundaries)
+void CompilerUtils::loadFromMemoryDynamic(
+	Type const& _type, bool _fromCalldata, bool _padToWordBoundaries, bool _keepUpdatedMemoryOffset)
```

Comment:
In my latest PR had a lot of back and forth with Gavin regarding how breaking a function to multiple lines should be. Here is his comment that shed a light on it. https://github.com/ethereum/cpp-ethereum/pull/1169#issuecomment-76964724

So I suppose if you really want to break it you should add each argument to its own line.